randovania/data/json_data/prime1.txt

```diff
@@ -2059,10 +2149,20 @@ Asset id: 3466748899
 > Dock to Frigate Crash Site; Heals? False
   > Frigate Crash Site/Dock to Overgrown Cavern
       Trivial
+  > Overgrown Cavern/Pickup (Missile Expansion)
+      Trivial
```

Comment:
This room requires Morph Ball on both sides to reach the pickup and either door.